core/src/main/java/nl/nn/adapterframework/stream/Message.java

```diff
@@ -395,6 +398,14 @@ public String asString(String defaultCharset) throws IOException {
 		if (request instanceof String) {
 			return (String)request;
 		}
+		if(request instanceof Document) {
```

Comment:
How would the request become a document? There is no constructor for that.

Reply:
Message.asMessage(Object)